src/main/view/screen/SettingsPanel.java

```diff
@@ -1,32 +1,392 @@
+			setCommandButtonVisibility(true);
+			g.setFont(getAssets().getFont(3));
+		    g.setFill(Color.WHITE);
+		    g.setEffect(ds);
+		    if (currentMode == SettingsEnum.HUMAN)
+		    	g.fillText("Human Controls", 15, 100);
+		    else
+		    	g.fillText("Panda Controls", 15, 100);
+		    g.setFont(getAssets().getFont(1));
+		    drawCommandLabels(g);
+		    g.setEffect(null);
+		}
+
+		private void drawCommandLabels(GraphicsContext g) {
+			int multiplier = 0;
+		    g.fillText("Mode Modifier", 15, 140 + SPACING  * multiplier++);
+		    g.fillText("Mode/Command Forward", 15, 140 + SPACING * multiplier++);
+		    g.fillText("Mode/Command Backward", 15, 140 + SPACING * multiplier++);
+		    g.fillText("Type/Instance Forward", 15, 140 + SPACING * multiplier++);
+		    g.fillText("Type/Instace Backward", 15, 140 + SPACING * multiplier++);
+		    g.fillText("Select Item", 15, 140 + SPACING * multiplier++);
+		    g.fillText("Show Unit Overview", 15, 140 + SPACING * multiplier++);
+		    g.fillText("Show Structure Overview", 15, 140 + SPACING * multiplier++);
+		    g.fillText("End Turn", 15, 140 + SPACING * multiplier++);
+		    g.fillText("Move North", 15, 140 + SPACING * multiplier++);
+		    g.fillText("Move North-East", 15, 140 + SPACING * multiplier++);
+		    g.fillText("Move South-East", 15, 140 + SPACING * multiplier++);
+		    g.fillText("Move South", 15, 140 + SPACING * multiplier++);
+		    g.fillText("Move South-West", 15, 140 + SPACING * multiplier++);
+		    g.fillText("Move North-West", 15, 140 + SPACING * multiplier++);
+		    g.fillText("Center Camera", 15, 140 + SPACING * multiplier++);
+		}
+
+		private void drawTopBar(GraphicsContext g) {
+		      g.drawImage(getAssets().getImage("GUI_MAP_BAR"), 0, 0);
+		      g.setFont(getAssets().getFont(2));
+		      g.setFill(Color.WHITE);
+		      g.setEffect(ds);
+		      g.fillText("Settings", 6, 35);
+		      g.setEffect(null);
+		}
+		
+		private void setCommandButtonVisibility(boolean visibility) {
+			if(visibility) {
+				modeModifier.setVisible(true);
+			} else {
+				modeModifier.setVisible(false);
+			}
+			for (Button button : controlButtons) {
+				button.setVisible(visibility);
+			}
+		}
+		
+		//Updates What's Displayed To What's in the Arrays
+		private void updateControlDisplay() {
+			if (currentMode == SettingsEnum.HUMAN) {
+				modeModifier.setValue(humanControls[0][1]);
+				for (int i = 1; i < humanControls.length; i++) {
+					controlButtons.get(i - 1).setText(humanControls[i][1]);
+				}
+			} else if (currentMode == SettingsEnum.PANDA) {
+				modeModifier.setValue(pandaControls[0][1]);
+				for (int i = 1; i < pandaControls.length; i++) {
+					controlButtons.get(i - 1).setText(pandaControls[i][1]);
+				}
+			}
+		}
+		
+		//Saves What is Visible In The Array
+        private void saveCurrentSelections() {
+			if (currentMode != SettingsEnum.GENERAL) {
+				if (currentMode == SettingsEnum.HUMAN) {
+					humanControls[0][1] = modeModifier.getValue();
```

Comment:
[0][1] is the Control Shift Alt. That has to be handled separately from everything else.